src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2465,8 +2465,26 @@ def create_managed_ssl_cert(cmd, resource_group_name, name, hostname, slot=None)
     location = webapp.location
     easy_cert_def = Certificate(location=location, canonical_name=hostname,
                                 server_farm_id=server_farm_id, password='')
-    return client.certificates.create_or_update(name=hostname, resource_group_name=resource_group_name,
-                                                certificate_envelope=easy_cert_def)
+
+    try:
+        return client.certificates.create_or_update(name=hostname, resource_group_name=resource_group_name,
+                                                    certificate_envelope=easy_cert_def)
+    except DefaultErrorResponseException as ex:
+        if ex.response.status_code == 202:
+            retry_times = 12
+            retry_delay = 5
+            while retry_times > 0:
```

Comment:
this I would check to see while the status of the GET response is not 200.

Reply:
Changed